README.md

```diff
@@ -132,6 +132,9 @@ accelerate launch -m axolotl.cli.inference examples/openllama-3b/lora.yml \
   docker compose up -d
   ```
 
+>[!Tip]
+> If you want to debug axolotl or prefer to use Docker as your development environment, see the [debugging guide's section on Docker](docs/debugging.md#debugging-with-docker).
+
```

Comment:
Should this section be put under the "Contributing section"?

Reply:
I think lots of times people use docker to debug stuff and they may not think of contributing 

At least all the questions I got were people wanting to debug but not necessarily contribute 

We take for granted that people want to open PRs so I put this here to get people further into the funnel of debugging their issues